ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java

```diff
@@ -67,31 +72,31 @@ public UserDto userUpdate(UserDto userDto) throws IOException, InterruptedExcept
         return result;
     }
 
-    public UserDto userCreate(NewUserDto newUserDto) throws InterruptedException, IOException {
-        UserDto userDto;
-        String resultString;
-        String userCreateAddress = PropertyManager.getProperty("backend.user_create");
-        log.debug("Connecting remote host {}", backendHost + userCreateAddress);
-        request = HttpRequest.newBuilder(URI.create(backendHost + userCreateAddress))
-                .header("Content-Type", "application/json")
-                .POST(HttpRequest.BodyPublishers.ofString(JsonMapper.toJson(newUserDto)))
+    public UserDto userCreate(String userName) throws InterruptedException, IOException {
+        log.debug("Creating a new user: {}, url address: {}", userName, userCreateAddress);
+        var request = HttpRequest.newBuilder(URI.create(userCreateAddress + userName))
+                .header(CONTENT_TYPE, APPLICATION_JSON)
+                .GET()
                 .build();
-        HttpResponse<String> response = null;
+        HttpResponse<String> response;
         try {
             response = client.send(request, HttpResponse.BodyHandlers.ofString());
         } catch (IOException e) {
+            log.error("User creation has failed: {}", e.getMessage());
             throw new HttpClientIOException("IOException to remote address " + userCreateAddress);
         }
-        resultString = response.body();
-        userDto = JsonMapper.fromJson(resultString, UserDto.class);
-        return userDto;
+        if (response.statusCode() > 399) {
+            throw new HttpClientIOException("IOException to remote address " + userCreateAddress);
```

Comment:
If we have response - there is no problem with connection. Its some kind of backend error. Ill make BackendErrorException